build.sh

```diff
@@ -82,51 +83,79 @@ function packageJmc() {
     echo "$(date +%T) jetty server up and running"
 
     popd 1> /dev/null || {
-        echo "could not go to project root directory"
+        err_log "could not go to project root directory"
         exit 1
     }
     pushd core 1> /dev/null || {
-        echo "directory core not found"
+        err_log "directory core not found"
         exit 1
     }
 
     echo "$(date +%T) installing core artfacts - logging output to ${installLog}"
     mvn clean install --log-file "${installLog}"
 
     popd 1> /dev/null || {
-        echo "could not go to project root directory"
+        err_log "could not go to project root directory"
         exit 1
     }
     echo "$(date +%T) packaging jmc - logging output to ${packageLog}"
     mvn package --log-file "${packageLog}"
 
-    echo "You can now run jmc by calling ${BASEDIR}/products/org.openjdk.jmc/linux/gtk/x86_64/JDK\ Mission\ Control/jmc"
+    if [[ "${OSTYPE}" =~ "linux"* ]]; then
+        echo "You can now run jmc by calling ${BASEDIR}/products/org.openjdk.jmc/linux/gtk/x86_64/JDK\ Mission\ Control/jmc"
```

Comment:
Perhaps refer to --run? You can now run jmc by calling this script with --run, or by calling...